inspirehep/modules/disambiguation/core/ml/models.py

```diff
@@ -25,14 +25,40 @@
 from __future__ import absolute_import, division, print_function
 
 import csv
+import json
 import pickle
+from scipy.special import expit
 
 import numpy as np
+from sklearn.ensemble import RandomForestClassifier
 from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.pipeline import FeatureUnion
 from sklearn.pipeline import Pipeline
 from sklearn.svm import LinearSVC
 
+from beard.similarity import CosineSimilarity
+from beard.similarity import ElementMultiplication
+from beard.similarity import EstimatorTransformer
+from beard.similarity import PairTransformer
+from beard.similarity import StringDistance
```

Comment:
you can also do
```
from beard.similarity import (CosineSimilarity,
                              ElementMultiplication,
                              #...
                              StringDistance)
```
and so on to avoid the wall of text.